tests/manage/pv_services/test_rwo_pvc_fencing_unfencing.py

```diff
@@ -534,6 +536,7 @@ def verify_multi_attach_error(self, pod_list):
 
         return True
 
+    @skipif_bm
     @tier4a
```

Comment:
@ebenahar we should skip tier4c tests as well for the same reason.

Reply:
done